modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/batch/returns.batch-engine-data.json

```diff
@@ -0,0 +1,126 @@
+{
+	"configuration": {
+		"className": "com.liferay.object.admin.rest.dto.v1_0.ObjectDefinition",
+		"multiCompany": true,
+		"parameters": {
+			"containsHeaders": "true",
+			"createStrategy": "UPSERT",
+			"featureFlag": "LPD-9361",
+			"importStrategy": "ON_ERROR_FAIL",
+			"taskItemDelegateName": "DEFAULT",
+			"updateStrategy": "UPDATE"
+		},
+		"userId": 0,
+		"version": "v1.0"
+	},
+	"items": [
+		{
+			"enableCategorization": true,
+			"externalReferenceCode": "L_RETURN",
```

Comment:
I don't know what Brian will say, but Return sounds too generic. Maybe L_COMMERCE_ORDER_RETURN? This is just a suggestion, do what you want.